python/tank/authentication/interactive_authentication.py

```diff
@@ -94,7 +96,7 @@ class SessionRenewal(object):
     _renew_session_internal_lock = threading.Lock()
 
     # List of possible states for session renewal.
-    WAITING, CANCELLED, SUCCESS = range(3)
+    WAITING, CANCELLED, SUCCESS = list(range(3))
```

Comment:
Why the range import at the top and no need to wrap in a list.

Reply:
✅ import is because sgsix does some remapping between range / xrange for cross-compat.  In our use here it's not necessary.